.travis.yml

```diff
@@ -32,6 +32,9 @@ matrix:
       install:
         - mdbook --help || cargo install mdbook --force
       script:
+        - cargo +nightly generate-lockfile -Z minimal-versions
+        - cargo check --tests
+        - cargo +nightly generate-lockfile
```

Comment:
Our "best practices" I think for this was to set this on a new travis matrix entry with the minimum supported Rust version, right? Should that be done here?

I'm somewhat hesitant to do this for Cargo though in the sense that we don't really have a minimum supported Rust version, it's always stable and forward for Cargo.

Reply:
Ya I was figuring that since we don't have a MRV we could just combine it.